client/lib/plans/constants.js

```diff
@@ -621,7 +628,13 @@ export const FEATURES_LIST = {
 		),
 		getStoreSlug: () => 'no-adverts/no-adverts.php'
 	},
-
+	[ FEATURE_REPUBLICIZE ]: {
+		getSlug: () => FEATURE_REPUBLICIZE,
+		getTitle: () => i18n.translate( 'Targeted Social Media' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **30** times:
`translate( 'Social Media' )` *ES Score: 7.30*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).